### ISSUE TITLE
High CPU Load

### ISSUE THREAD

**Issue by befedo** [akiyosi/goneovim#88]
Hi there,

When going from fab586f to 515efbb goneovim causes a high CPU load, even with no `settings.toml`.

QT settings are:
```{bash}
$ qtsetup check
INFO[0000] running: 'qtsetup check linux' [docker=false] [vagrant=false] 
INFO[0000] GOOS:                        'linux'         
INFO[0000] GOARCH:                      'amd64'         
INFO[0000] GOVERSION:                   'go1.13.5'      
INFO[0000] GOROOT:                      '/usr/lib/go'   
INFO[0000] GOPATH:                   *  '/home/user/go' 
INFO[0000] GOBIN:                       '/home/user/go/bin' 
INFO[0000] GOMOD:                       ''              
INFO[0000] QT_HASH:                     '2b818d970888dab6bbf6aa2702bc28c1e6fbdb83' 
INFO[0000] QT_API:                   *  '5.13.0'        
INFO[0000] QT_VERSION:               *  '5.14.0'        
INFO[0000] QT_DIR:                      '/home/user/Qt' 
INFO[0000] QT_STUB:                     'false'         
INFO[0000] QT_DEBUG:                    'false'         
INFO[0000] QT_QMAKE_DIR:                ''              
INFO[0000] QT_WEBKIT:                   'false'         
INFO[0000] QT_STATIC:                   'false'         
INFO[0000] QT_GEN_TSD:                  'false'         
INFO[0000] QT_GEN_OPENGL:               'false'         
INFO[0000] QT_GEN_QUICK_EXTRAS:         'false'         
INFO[0000] QT_RESOURCES_BIG:            'false'         
INFO[0000] QT_NOT_CACHED:               'false'         
INFO[0000] QT_DISTRO:                   'arch'          
INFO[0000] QT_PKG_CONFIG:            *  'true'          
INFO[0000] QT_DOC_DIR:                  '/usr/share/doc/qt' 
INFO[0000] QT_MISC_DIR:                 '/usr/lib/qt'
```

Has anyone other these issues?

Thanks for your time and best regards,
Befedo

**Comment by ur4ltz** [comment]
Hi!

Yes CPU load 98-100%

**Comment by akiyosi** [comment]
Hi :)

Thank you for this report.
I have identified the cause and fixed this in https://github.com/akiyosi/goneovim/commit/d872e51cbd0fcd1e46123fdb1e34b290f4c0a790
I will plan to include this fix for v0.4.3

**Comment by befedo** [comment]
> Hi :)
> 
> Thank you for this report.
> I have identified the cause and fixed this in [d872e51](https://github.com/akiyosi/goneovim/commit/d872e51cbd0fcd1e46123fdb1e34b290f4c0a790)
> I will plan to include this fix for v0.4.3

Wow that was fast!

Thank you so much for your effort, goneovim is an awesome looking gui ...

Best regards,
Befedo
